dll/win32/shell32/brfolder.cpp

```diff
@@ -285,56 +312,31 @@ static void
 BrFolder_Expand(
     BrFolder *info,
     IShellFolder *lpsf,
-    PCIDLIST_ABSOLUTE pidl,
-    HTREEITEM hParent,
-    IEnumIDList* pEnum)
+    PCIDLIST_ABSOLUTE pidlFull,
+    HTREEITEM hParent)
```

Comment:
May be worth adding SAL2 annotations here too.

Reply:
Added.